### PR TITLE
Timeseries search fixes

### LINKED ISSUE #266: Add option to hide root asset select in TimeSeriesSearch

### PR DESCRIPTION
**Pull request by follan**
Fixes #266

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-270.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx [hunk 1]
```diff
@@ -69,9 +74,11 @@ export class TimeseriesSearch extends React.Component<
 > {
   static defaultProps = {
     selectedTimeseries: [],
+    strings: {},
     filterRule: undefined,
     hideSelected: false,
     allowStrings: false,
+    hideRootAssetSelect: false,
```
Comment (mrtraser) [inline src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Maybe better to have the same prop value as for Search component (set to `false` by default)?
```typescript tsx
<TimeseriesSearch 
  ...
  rootAssetSelect={true}
/>
```

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-270.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-270.gearboxjs.preview.cogniteapp.com

**Pull request merged**